src/domain/event/CreateEventPage.tsx

```diff
@@ -35,11 +36,16 @@ const CreateEventPage: React.FC = () => {
         console.log('TODO: Submit event form with values: ', values);
       }}
       validationSchema={createEventValidationSchema}
-      validateOnBlur={true}
-      validateOnChange={false}
       validateOnMount
     >
-      {({ values: { type } }) => {
+      {({ values: { type, ...restValues } }) => {
+        const saveDraft = () => {
+          console.log('TODO: Save draft with values: ', {
```

Comment:
is it intentional that this one is .log and not .alert? When testing the UI it wasn't clear what happened because I didn't have console open

Reply:
That is just something that came to my mind, not major issue

Reply:
This is actually something that will be solved on next PR which will implement saving event feature